ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml

```diff
@@ -0,0 +1,89 @@
+periodics:
+- agent: kubernetes
+  cluster: build01
+  cron: 0 6 25 10 *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: pit-7.4.x
+    org: jboss-eap-qe
+    repo: openshift-eap-tests
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: eap-ocp4.15-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.15"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.15-lp-interop-eap-74-xp-interop-aws
```

Comment:
I think you may be missing a Slack config here, unless you don't need Slack pass/fail notifications on this job.

Reply:
Yeah @calebevans , we are left with the addition of `reporter_config` to add the slack channel configuration. 
@oharan2 , once you are back please add
`reporter_config:
    slack:
      channel: '#eap-ocp-ci-results'
      job_states_to_report:
      - success
      - failure
      - error
      report_template: '{{if eq .Status.State "success"}} :success: Job *{{.Spec.Job}}*
        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> {{else}} :dumpster-fire:
        Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
        logs> {{end}}'`
I already confirmed it from PQE.